nist_csf_v2/function_de.pp

```diff
@@ -0,0 +1,250 @@
+    benchmark.nist_csf_v2_de_cm_09
+  ]
+}
+
+benchmark "nist_csf_v2_de_cm_01" {
+  title       = "DE.CM-01"
+  description = "Networks and network services are monitored to find potentially adverse events."
+  children = [
+    control.cloudtrail_multi_region_trail_enabled,
+    control.cloudtrail_trail_enabled,
+    control.cloudtrail_trail_integrated_with_logs,
+    control.cloudtrail_trail_logs_encrypted_with_kms_cmk,
+    control.cloudtrail_trail_validation_enabled,
+    control.cloudwatch_alarm_action_enabled,
+    control.elb_application_classic_lb_logging_enabled,
+    control.log_metric_filter_network_acl,
+    control.log_metric_filter_network_gateway,
+    control.log_metric_filter_route_table,
+    control.log_metric_filter_security_group,
+    control.log_metric_filter_vpc,
+    control.vpc_flow_logs_enabled
+  ]
+}
+
+benchmark "nist_csf_v2_de_cm_02" {
+  title       = "DE.CM-02"
+  description = "The physical environment is monitored to find potentially adverse events."
+  children = [
+    control.autoscaling_group_multiple_az_configured,
+    control.cloudwatch_alarm_action_enabled,
+    control.ec2_instance_detailed_monitoring_enabled,
+    control.rds_db_instance_and_cluster_enhanced_monitoring_enabled,
+    control.rds_db_instance_multiple_az_enabled
+  ]
+}
+
+benchmark "nist_csf_v2_de_cm_03" {
+  title       = "DE.CM-03"
+  description = "Personnel activity and technology usage are monitored to find potentially adverse events."
+  children = [
+    control.cloudtrail_multi_region_trail_enabled,
+    control.cloudtrail_trail_enabled,
+    control.cloudtrail_trail_integrated_with_logs,
+    control.cloudtrail_trail_logs_encrypted_with_kms_cmk,
+    control.cloudtrail_trail_validation_enabled,
+    control.iam_user_console_access_mfa_enabled,
+    control.iam_user_in_group,
+    control.iam_user_mfa_enabled,
+    control.log_metric_filter_console_authentication_failure,
+    control.log_metric_filter_console_login_mfa,
+    control.log_metric_filter_iam_policy,
+    control.log_metric_filter_unauthorized_api
+  ]
+}
+
+benchmark "nist_csf_v2_de_cm_06" {
+  title       = "DE.CM-06"
+  description = "External service provider activities and services are monitored to find potentially adverse events."
+  children = [
+    control.cloudwatch_cross_account_sharing,
+    control.log_metric_filter_bucket_policy,
+    control.log_metric_filter_cloudtrail_configuration,
+    control.log_metric_filter_config_configuration,
+    control.sns_topic_policy_prohibit_cross_account_access
+  ]
+}
+
+benchmark "nist_csf_v2_de_cm_09" {
+  title       = "DE.CM-09"
+  description = "Computing hardware and software, runtime environments, and their data are monitored to find potentially adverse events."
+  children = [
+    control.cloudwatch_alarm_action_enabled,
+    control.dynamodb_table_in_backup_plan,
+    control.dynamodb_table_point_in_time_recovery_enabled,
+    control.dynamodb_table_protected_by_backup_plan,
```

Comment:
Shall we include `backup/lifecycle` controls in benchmark `DE.CM-09` as the benchmark has a hardware monitoring context?